content/en/docs/reference/config/analysis/ist0102/index.md

```diff
@@ -0,0 +1,13 @@
+---
+title: NamespaceNotInjected
+layout: analysis-message
+---
+
+This message occurs when you have a namespace that is missing the
+[annotation](/docs/reference/config/annotations/) to indicate whether the
+namespace is auto-injected, for example `sidecar.istio.io/inject`.
+
+To resolve this problem, always use an annotation to explicitly declare whether
+or not you want the namespace to be auto-injected. We strongly recommend
+explicitly defining desired injection behavior. Forgetting to annotate a
+namespace is a common cause of errors.
```

Comment:
Should supply a command-line showing how to apply the annotation to a specific namespace, and to all namespaces.

Reply:
Fixed for the singular case. However, I have been unable to find an example of how to apply an annotation to multiple namespaces at once. If anyone has the example, send it to me and I will add it to the docs.